app/views/ops/_all_tabs.html.haml

```diff
@@ -14,6 +14,7 @@
         = miq_tab_content("settings_evm_servers", @sb[:active_tab]) do
           = render :partial => "settings_evm_servers_tab"
     - elsif selected?(x_node, "svr")
+      = render :partial => "layouts/flash_msg"
```

Comment:
Sounds like we may want to level-up this even more eventually.. but yup, matches the removals for now :)